main.go

```diff
@@ -21,20 +21,34 @@ func main() {
 	defer recoverPanic()
 	ctx := context.Background()
 	//Initialize the config
-	if err := config.InitConfig(); err != nil {
+
+	if err := config.InitConfig("mongodb"); err != nil {
```

Comment:
dont modify the `InitConfig`  method, instead set default value to  true for either mongo/postgres in config file